tests/test_runtime_checks.py

```diff
@@ -0,0 +1,12 @@
+from basedtyping.runtime_checks import is_subclass
+
+# pylint:disable=no-self-use
+
+
+class TestIsSubclass:
```

Comment:
Why is this a class

Reply:
Just to group the tests

Reply:
there was another runtime check in there so it was 2 classes, but i removed it